concent_api/core/views.py

```diff
@@ -32,11 +33,7 @@
 @log_communication
 def send(_request: HttpRequest, client_message: Message, client_public_key: bytes) -> Union[Message, HttpResponse]:
     assert isinstance(client_public_key, bytes) or client_public_key is None
-    if client_public_key is not None:
```

Comment:
Are we sure now key is always present? If yes please also remove it from `assert` above.

Reply:
Right, it sometimes is None. Fixed and passed as `Optional`

Reply:
:+1: for see this. Sometimes it is `None`. Fixed and passed as Oprional